.azuredevops/deploy-platform-bootstrap-env.yaml

```diff
@@ -15,6 +15,13 @@ parameters:
       - Monitoring - Workspace
       - Policy Assignments
       - Network
+  - name: Federated
+    displayName: "Use Federated Credential"
+    type: string
+    default: "False"
+    values:
```

Comment:
change the type for this parameter to be boolean and remove the values please

Reply:
Added Boolean parameter and removed the values